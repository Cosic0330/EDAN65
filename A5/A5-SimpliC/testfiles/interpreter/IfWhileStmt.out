New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
New put: Variable: 0_a Value: 10
{0_a=10, returnValue=0, return=0}
New get: Variable: 0_a Value: 10
New get: Variable: return Value: 0
New get: Variable: 0_a Value: 10
New put: Variable: 0_a Value: 20
{0_a=20, returnValue=0, return=0}
New get: Variable: return Value: 0
New put: Variable: 0_a Value: 15
{0_a=15, returnValue=0, return=0}
New get: Variable: return Value: 0
New get: Variable: 0_a Value: 15
New get: Variable: 0_a Value: 15
New put: Variable: 0_a Value: 30
{0_a=30, returnValue=0, return=0}
New get: Variable: return Value: 0
New put: Variable: 4_b Value: 0
{0_a=30, returnValue=0, 4_b=0, return=0}
New get: Variable: 4_b Value: 0
New get: Variable: return Value: 0
New get: Variable: 4_b Value: 0
New get: Variable: 4_b Value: 0
New put: Variable: 4_b Value: 1
{0_a=30, returnValue=0, 4_b=1, return=0}
New get: Variable: 4_b Value: 1
New get: Variable: return Value: 0
