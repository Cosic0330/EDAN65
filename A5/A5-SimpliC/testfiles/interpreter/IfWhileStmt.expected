New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
New put: Variable: a Value: 10
{a=10, returnValue=0, return=0}
New get: Variable: a Value: 10
New get: Variable: return Value: 0
New get: Variable: a Value: 10
New put: Variable: a Value: 20
{a=20, returnValue=0, return=0}
New get: Variable: return Value: 0
New put: Variable: a Value: 15
{a=15, returnValue=0, return=0}
New get: Variable: return Value: 0
New get: Variable: a Value: 15
New get: Variable: a Value: 15
New put: Variable: a Value: 30
{a=30, returnValue=0, return=0}
New get: Variable: return Value: 0
New put: Variable: b Value: 0
{a=30, b=0, returnValue=0, return=0}
New get: Variable: b Value: 0
New get: Variable: return Value: 0
New get: Variable: b Value: 0
New get: Variable: b Value: 0
New put: Variable: b Value: 1
{a=30, b=1, returnValue=0, return=0}
New get: Variable: b Value: 1
New get: Variable: return Value: 0
